src/main/java/org/opensearch/sdk/ActionExtension.java

```diff
@@ -1,191 +0,0 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- */
-
-package org.opensearch.sdk;
```

Comment:
This should have been moved into `org.opensearch.sdk.api` with all the rest.  Where did it go?